GSA/digitalgov-pra
# Reduce image sizes and script loading performance

## Summary
Improve critical performance metrics identified by Lighthouse report.

### Preview

[Link to Preview](https://federalist-ba11378c-de49-4b18-b3c7-6dde8c4aa829.sites.pages.cloud.gov/preview/gsa/digitalgov-pra/nl-improve-performance/)

### Solution

1. Reduce identifier OMB logo image size — de0f86765cffdd746bf068d721453aee18f96b63
2. Set `width` & `height` on home page images to [reduce CLS](https://web.dev/optimize-cls/?utm_source=lighthouse&utm_medium=devtools#images-without-dimensions) — 4522c747c8daf03c64b8e66a39f88180bf1542b0
3. Removed font-awesome icons that are used only once — 4a66fd5004070969f2bc21cad09ec79029410978


### How To Test

1. Visit homepage
2. Run Lighthouse and should not see warnings for image sizes and serving large images that are scaled down

---

### Dev Checklist
